.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     - id: end-of-file-fixer
     - id: trailing-whitespace
     - id: flake8
-      args: [--max-line-length=100]
+      args: ["--max-line-length=100", "--per-file-ignores=files/packit.wsgi:F401"]
```

Comment:
The line starts to be a bit long, what about splitting it (and removing the quotation marks)? This works for me:

```yaml
- id: flake8
      args: 
        - --max-line-length=100
        - --per-file-ignores=files/packit.wsgi:F401
```

Reply:
Ok, why not.